concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1122,6 +1124,7 @@ jobs:
           - get: datadog-tfstate
           - get: paas-rubbernecker
           - get: paas-compose-broker
+            version: "04d277d7c20d8674df40d355a2c1a6a109b15d5a"
```

Comment:
We've found this method of version pining problematic in the past. It won't work for new deployments (or newly pushed pipelines) if this commit isn't the HEAD of the branch this resource is pointing at. The reason for this is that this feature can only select from versions that the resource has seen. For a new deployment, this list will only include the HEAD commit, and not any previous ones.

We should change this to instead pin the resource to a specific tag.